syft/frameworks/torch/mpc/primitives.py

```diff
@@ -264,3 +265,34 @@ def build_separate_triples(n_party: int, n_instances: int, **kwargs) -> list:
             return primitives_worker
 
         return build_separate_triples
+
+    def __get_next_elem(self, generator, shape=(1,), size=2 ** 32):
```

Comment:
low level details and helper functions should come after abstract concepts, I would suggest moving it down after the generation functions

Reply:
Added it at the end :)